src/containers/footer-lists/footer-lists.js

```diff
@@ -29,11 +29,9 @@ const FooterLists = ({ language = LANGUAGE }) => {
     }
   };
 
-  const CategoriesView = () => {
-    if (!categories) {
-      return null;
-    }
-    return categories.map(({ _id, name }) => (
+  const categoriesList = !categories
```

Comment:
Usually, it is better to have positive case first `categories ? categories.map(...) : null`

Reply:
Changed